### PR TITLE
add 4 icons

### PR DESCRIPTION
**Pull request by piecesuger**
# Description

Add icons for  GKD, Copymanga, JMComic2, InstallerX
### Icons added
* GKD (`li.songe.gkd`)
* Copymanga (`com.copymanga.app`)
* JMComic2 (`com.jiaohua_browser`)
* InstallerX (`com.rosan.installer.x`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

File: svgs/_k.svg
Comment (x9136) [inline svgs/_k.svg:1]: Screenshot 2023-10-31 at 12 17 11
It's too bold in the center. That needs to be dealt with somehow. This is one of the variations of what I came up with.

**Review by x9136 — CHANGES_REQUESTED** [review]

File: svgs/_k.svg
Comment (piecesuger) [reply svgs/_k.svg:1]: Done.

Comment (x9136) [reply svgs/_k.svg:1]: Screenshot 2023-10-31 at 14 46 19
Visually it's a good option, but you've shifted it 4px lower than it needs to be.

Comment (piecesuger) [reply svgs/_k.svg:1]: Got it. Fixed.

**Review by x9136 — APPROVED** [review]

**Pull request merged**